news/2018-04-19-twc-ro16-recap.md

```diff
@@ -0,0 +1,93 @@
+
+Canada and Taiwan made Malaysia and Australia meet the same fate as Brazil and the Maustraysian duo will also be playing in the losers 
+bracket from now on.
+
+Things got really interesting in the match between France and Indonesia. The losses of France usually came down to a single misplay, and 
+they eventually took a point on the FreeMod [t+pazolite - Elder Dragon Legend (ft. Kabocha)(https://osu.ppy.sh/beatmapsets/761951#taiko/1602056) despite the 2 mod advantage of Indonesia. Overall though, **Indonesia will take their revenge on France 5-1**, the same score they lost by last year.
+
+Only one Argentinian player woke up for their match, resulting in a **win by default for Japan.**
+
+South Korea edged out Poland on every match they played, but that didn't stop CreepyDuck from pulling off this crazy score on Viden:
+![](/wiki/shared/news/2018-04-19-twc-ro16-recap/viden-poland-south-korea.jpg)
+
+Hong Kong missing one of their three players put them in a tough spot against Chile with no replacement players. The accuracy monster sing216 lived up to their reputation by getting 5 scores above 99,9%, 3 of which were SS ranks. The other Hong Kong player, Faputa, couldn't quite match that but still exceeded the expectations of many and helped put Hong Kong on the board. **It ended 5-1 in favour of the Chileans.** 
+
+The last match of the day was also arguably the most even one, United Kingdom vs Spain.  This one could have been significantly closer, but with the small score differences all going the way of Spain, they clinched the match 5-1. However, with the dedication United Kingdom displayed on the memorization map [Black Hole - Pluto](https://osu.ppy.sh/beatmapsets/599679/#taiko/1599257), **teams in the losers bracket should not take them lightly.**
+![](/wiki/shared/news/2018-04-19-twc-ro16-recap/pluto-uk-fcs.jpg)
+
+
+### Round Of 16 Mappool
+
+**The picks were evenly spread out this week, more so than in the Group Stage anyway.** Across 7 matches, every map was played at least once except for tiebreaker. Teams kept it simple by banning out technical maps a lot, such as [onoken - Viden(https://osu.ppy.sh/beatmapsets/708312/#taiko/1497491) (Hidden) and [Virt - Staring at my Spaceship](https://osu.ppy.sh/beatmapsets/57145/#taiko/766477) (HardRock). 
+
+
+### Quarterfinals Preview
+
+In the Quarterfinals winners bracket, we'll be seeing: Japan vs. Chile, South Korea vs. Indonesia, Spain vs. USA and Taiwan vs. Canada.
+**The losers of these matches will drop down to the losers bracket.** Speaking of the losers bracket, the matches we'll be seeing there this weekend are: Argentina vs. Hong Kong, Poland vs. France, United Kingdom vs. Brazil and Australia vs. Malaysia. 
```

Comment:
is this paragraph supposed to be by itself or is it part of the line above, if the latter, merge it with the one above